sidhariyal/star_names.fab

```diff
@@ -0,0 +1,1181 @@
+ 87294|_("Vanant") 11
+ 86670|_("மூலம் 3/குருகு") 11
+ 86670|_("Girtab") 20,38
+ 82514|_("கேட்டை 3/துழங்கொளி") 12
+ 82514|_("Tali al Shaulah I") 11
+ 82545|_("Pipirima") 12
+ 82545|_("Tali al Shaulah II") 11
+ 78265|_("அனுசம் 3/முடப்பணை") 12
+ 78265|_("Nur") 11
+ 78104|_("Iklil") 12
+ 78104|_("Iolil") 11
+ 78933|_("Jabhat Acrabi I") 11
+ 78933|_("Jabhat al Akrab") 38
+ 78990|_("Jabhat Acrabi II") 11
+ 87261|_("Fuyue") 12
+ 82396|_("கேட்டை 2/துழங்கொளி") 12
+ 82396|_("Wei") 38
+ 83547|_("Rapeto") 36
+ 79431|_("Sharjah") 36
+# Sculptor (Scl)
+  3479|_("Cocibolca") 36
+# Serpens (Ser)
+ 77070|_("Unukalhai") 1,2,6,11,12,13,18,37,38
+ 77070|_("Cor Serpentis") 18,37
+ 77070|_("Unuk al Hay") 18,23
+ 77070|_("Unuk Elhaia") 23
+ 92946|_("Alya") 1,2,6,11,12,18,38
+ 77233|_("Chow") 2,3,37
+ 77233|_("Nasak Shamiya I") 11
+ 78072|_("Nasak Shamiya II") 11
+ 76276|_("Nasak Yamani I") 11
+ 76276|_("Tsin") 38
+ 77622|_("Nasak Yamani II") 11
+ 92037|_("Tweedledee and Tweedledum") 22
+ 77450|_("Gudja") 12
+ 90004|_("Alasia") 36
+ 92895|_("Kaveh") 36
+ 89962|_("Alava") 38
+# Sextans (Sex)
+ 48711|_("Bibhā") 36
+# Taurus (Tau)
+ 21421|_("ரோகிணி/சகடு") 1,2,6,11,12,13,23,37,38
+ 21421|_("Cor Tauri") 37
+ 21421|_("Parilicium") 37
+ 25428|_("Elnath") 1,2,3,6,11,12
+ 25428|_("El Nath")
+ 25428|_("Nath") 23
+ 25428|_("Al Nath") 38
+ 20889|_("Ain") 1,2,6,11,12,37,38
+ 20889|_("Oculus Boreus") 37
+ 17702|_("கார்த்திகை 2/ஆரல்") 1,2,6,11,12,23,37,38
+ 17489|_("Celaeno") 1,2,6,11,12
+ 17499|_("கார்த்திகை 4/ஆரல்") 1,2,6,11,12,38
+ 17531|_("கார்த்திகை 6/ஆரல்") 1,2,6,11,12,37
+ 17573|_("கார்த்திகை 5/ஆரல்") 1,2,6,11,12,38
+ 17579|_("Asterope") 1,2,6,11,12
+ 17579|_("Sterope I")
+ 17608|_("கார்த்திகை 3/ஆரல்") 1,2,6,11,12
+ 17847|_("கார்த்திகை 1/ஆரல்") 1,2,6,11,12,37,38
+ 17851|_("Pleione") 1,2,6,11,12
+ 20205|_("Prima Hyadum") 12
+ 20205|_("Hyadum I") 2,11
+ 20205|_("Hyadum Primus") 38
+ 20455|_("Secunda Hyadum") 12
+ 20455|_("Hyadum II") 2,11
+ 25945|_("The Ruby Star")
+ 17588|_("Sterope II") 11
+ 20885|_("Hyadum III") 11
+ 20894|_("Chamukuy") 12
+ 20894|_("Hyadum IV") 11
+ 20635|_("Alkalbain III") 11
+ 20641|_("Alkalbain IV") 11
+ 20711|_("Alkalbain V") 11
+ 20250|_("Alkalbain I") 11
+ 20430|_("Alkalbain II") 11
+ 26451|_("Tianguan") 12
+ 26451|_("Alheka") 38
+ 21109|_("Hoggar") 36
+# Triangulum (Tri)
+  8796|_("Mothallah") 1,6,11,12,37
+  8796|_("Caput Trianguli") 18,37
+  8796|_("Methallah") 23,37
+  8796|_("Atria") 37
+  8796|_("Rasalmudallah") 37
+  8796|_("Rasalmothallah") 38
+ 10064|_("Mizan") 11
+# Triangulum Australe (TrA)
+ 82273|_("Atria") 1,6,11,12,18,38
+# Tucana (Tuc)
+116084|_("Poerava") 36
+  5529|_("Emiw") 36
+# Ursa Major (UMa)
+ 54061|_("சிவன்") 1,2,6,11,12,13,18,23,37,38
+ 54061|_("Dubb") 18,37
+ 54061|_("Ak") 37
+ 54061|_("Dubh") 18
+ 53910|_("முருகன்") 1,2,6,11,12,18,23,38
+ 53910|_("Mirak") 18
+ 58001|_("திருமால்") 1,2,6,11,12,18
+ 58001|_("Phad") 18,38
+ 58001|_("Phekda") 23
+ 59774|_("கிருட்டினன்") 1,2,6,11,12,18,23,37,38
+ 59774|_("கிருட்டினன்") 37

```

Comment:
If this is the same as the line above, just delete this line! The original file likely has two different spellings, one only used in Ref.37 (in addition to the first spelling variant).

Reply:
Noted with thanks, I'll incorporate it.